lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -463,7 +463,18 @@ def set_sort_zpos(self, val):
         self._sort_zpos = val
         self.stale = True
 
-    def set_3d_properties(self, zs, zdir):
+    def set_3d_properties(self, zs, zdir='z'):
+        """
+        Update the z values of the offsets. If there are more XY points in
+        the offsets than there are Z points then points without a Z offset
+        will not be drawn.
```

Comment:
:+1: that's definitely an improvement. My only question is what tense is the docstring speaking in? Since the draw will happen after this function is called I used `will not be` but `are` switches to the present tense which I found confusing. Not a big deal either way.